pages/success-banner/index.js

```diff
@@ -39,7 +75,10 @@ const Footer = () => (
     made with
     <img className="icon" src="/static/love.svg" />
     by
-    <img className="brand" src="/static/resin.png" />
+    <Link href="https://resin.io/"
+      label="Resin">
+      <img className="brand" src="/static/resin.png" />
```

Comment:
Given that this page is only intended to be displayed inside the Etcher webview, I guess it doesn't really make sense to add ALT attributes to the IMG tags?